src/ui/common/src/utils/shared.ts

```diff
@@ -36,6 +36,38 @@ export enum ExecutionStatus {
   Running = 'running',
 }
 
+export const stringToExecutionStatus = (status: string): ExecutionStatus => {
```

Comment:
This appears to be costly to maintain (as ideally we need to add for all enums). Curious if there's easier way to do this

Reply:
I needed to go with string in this situation as the Checks and Metrics related tables needed to know their ExecutionStatus to properly be rendered.

TableData only allows for strings, and numbers, things that are serializable. I could expand this to support arbitrary objects like ExecutionStatus, but I felt that this was a quicker fix and a better separation of concerns in the moment.